.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+## Checklist
+
+* [ ] I've referred to the the [Guidelines for contributing](https://github.com/amiaopensource/ffmprovisr/blob/gh-pages/readme.md#guidelines-for-contributing)
+* [ ] New recipe has been added to `recipes.txt`
```

Comment:
I am not sure about the second point. Would this not confuse a person making her/his first contribution?

I am happy to sync form time to time, as I did in the past. (There is a on-liner here, but I personally use a short Perl script to gather the data from GitHub, to generate the correct output for FFCommand_Engine and to install it directly at the right place.)

Reply:
Thanks for clarifying Reto - I'm happy to remove it. I wasn't really familiar with the recipes.txt protocol and saw a recent commit you made and so I added it in.

Reply:
I could add here a new document explaining what is FFCommand_Engine, how to compile its last version and how to import ffmprovisr’s recipe into it.